php/README.md

```diff
@@ -0,0 +1,73 @@
+# Learning PHP REST API Project
+
+This is a basic PHP project created for learning purposes. The project focuses on making REST API calls and implementing middleware for a simple web application.
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have PHP and composer and  installed on your machine. Furthermore it is also recommended to have NPM and Node.js installed.
+
+If not, follow the instructions for installing PHP regading the [PHP website](https://www.php.net/manual/en/install.php).
+Since the documentation on the official website can be a little bit overwhelming, especially for Windows,
+feel free to watch a [youtube video](https://www.youtube.com/watch?v=MPRLUd8Pmyo) which guides you through the PHP installation.
```

Comment:
perfect! I really like the readme, super clear. I will test it on my local machine this weekend.